src/listeners/messageCreate.ts

```diff
@@ -66,14 +66,42 @@ client.on("messageCreate", async message => {
 		return void (await message.crosspost())
 
 	// Delete non-stringURL messages in review-strings
-	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
-	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings")) {
-		if (!/(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi.test(message.content))
-			await message.delete()
-		else {
+	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/\w+\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
```

Comment:
This regex will not match if two URLs are next to each other due to a bug in JavaScript, (see why [here](https://stackoverflow.com/questions/1520800/why-does-a-regexp-with-global-flag-give-wrong-results/1520853#1520853)). The fix is literally just this
```suggestion
	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/\w+\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/i
```